corehq/apps/accounting/tasks.py

```diff
@@ -215,6 +215,14 @@ def remind_subscription_ending_30_days(based_on_date=None):
     send_subscription_reminder_emails(1, exclude_trials=False)
 
 
+@periodic_task(run_every=crontab(minute=0, hour=0))
+def remind_dimagi_contact_subscription_ending_40_days(based_on_date=None):
```

Comment:
Good catch! Copy pasted the signature from another function and forgot to remove this.